modules/cudacodec/include/opencv2/cudacodec.hpp

```diff
@@ -369,28 +372,30 @@ class CV_EXPORTS_W VideoReader
 
     /** @brief Sets a property in the VideoReader.
 
-    @param property Property identifier from cv::cudacodec::VideoReaderProps (eg. cv::cudacodec::PROP_DECODED_FRAME_IDX, cv::cudacodec::PROP_EXTRA_DATA_INDEX, ...)
+    @param propertyId Property identifier from cv::cudacodec::VideoReaderProps (eg. cv::cudacodec::PROP_DECODED_FRAME_IDX, cv::cudacodec::PROP_EXTRA_DATA_INDEX, ...).
     @param propertyVal Value of the property.
     @return `true` if the property has been set.
      */
     CV_WRAP virtual bool set(const VideoReaderProps propertyId, const double propertyVal) = 0;
 
     /** @brief Returns the specified VideoReader property
 
-    @param property Property identifier from cv::cudacodec::VideoReaderProps (eg. cv::cudacodec::PROP_DECODED_FRAME_IDX, cv::cudacodec::PROP_EXTRA_DATA_INDEX, ...)
+    @param propertyId Property identifier from cv::cudacodec::VideoReaderProps (eg. cv::cudacodec::PROP_DECODED_FRAME_IDX, cv::cudacodec::PROP_EXTRA_DATA_INDEX, ...).
+    @param propertyValOut Value for the specified property.
     @param propertyVal Optional value for the property.
-    @return Value for the specified property. Value -1 is returned when querying a property that is not supported.
+    @return `true` unless the property is not supported.
     */
-    CV_WRAP virtual int get(const VideoReaderProps propertyId, const int propertyVal = -1) const = 0;
+    CV_WRAP virtual bool get(const VideoReaderProps propertyId, double& propertyValOut, const int propertyVal = -1) const = 0;
```

Comment:
> CV_WRAP

It needs to add CV_OUT specifier:

```
CV_OUT double& propertyValOut
```

---

> const int propertyVal = -1

Perhaps we don't need this anymore.